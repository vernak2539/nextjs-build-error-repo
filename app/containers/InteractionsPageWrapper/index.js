import React from 'react';
import PropTypes from 'prop-types';

import AppWrapperContainer from '../AppWrapper';

const InteractionsPageWrapper = (props) => (
  <AppWrapperContainer isError={props.isError}>{props.children}</AppWrapperContainer>
);

InteractionsPageWrapper.propTypes = {
  children: PropTypes.node.isRequired,
  /* if the app is in error state, ie SHS is down */
  isError: PropTypes.bool,
};

InteractionsPageWrapper.defaultProps = {
  isError: false,
};

export default InteractionsPageWrapper;
