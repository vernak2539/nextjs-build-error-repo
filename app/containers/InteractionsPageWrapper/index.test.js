import React from 'react';
import { shallow } from 'enzyme';

import InteractionsPageWrapper from '.';

describe('InteractionsPageWrapper container', () => {
  it('renders as expected', () => {
    const container = shallow(<InteractionsPageWrapper>Child</InteractionsPageWrapper>);
    expect(container).toMatchSnapshot();
  });
});
