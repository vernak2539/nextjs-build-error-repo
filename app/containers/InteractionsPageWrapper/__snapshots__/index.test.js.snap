// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InteractionsPageWrapper container renders as expected 1`] = `
<Connect(withRouter(AppWrapper))
  isError={false}
>
  <Connect(AppBanner) />
  Child
</Connect(withRouter(AppWrapper))>
`;
